app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -339,6 +350,12 @@ public void onResume() {
             media = getArguments().getParcelable("media");
         }
 
+        media = detailProvider.getMediaAtPosition(index);
+
+        if(media != null && applicationKvStore.getBoolean("Nominating " + media.getDisplayTitle(), false)) {
```

Comment:
The KV store is used to keep track of what files are being nominated, if I understand correctly?
How about a `final static` String rather than this inline String which seems to be duplicated several times below?
Also, does `getDisplayTitle` give the best identifier? We need something that does not change and is unique. I don't remember exactly but `getDisplayTitle` might return the same for several different images (screenshot showing pics I uploaded yesterday, same label displayed for two different pictures), and might also change if I change the locale?
![adb](https://user-images.githubusercontent.com/99590/120192227-df579600-c255-11eb-83be-6a95625386ea.png)

Reply:
Thanks for the review, I will make the required changes and I got your point. How about we use ImageUrl instead of display title, I think that will be unique for each image.

Reply:
Yes good idea!